tests/unit/test_pagerank.c

```diff
@@ -106,6 +106,7 @@ void test_pagerank() {
 	}
     
 	rm_free(ranking);
+	GrB_free(&A);
```

Comment:
it was because on the suppression we added it back than because we didn't have the stacktrace from from when the leak now I found the leak and updated the suppression